### PR TITLE
Fix page range rendering

### PR DESCRIPTION
**Pull request by DerDrodt**
This PR (together with one for citationberg) fixes two citeproc tests (those for Chicago/Chicago16 page ranges).

### CODE REVIEW

File: src/csl/rendering/mod.rs [hunk 1]
```diff
@@ -85,6 +85,11 @@ impl RenderCsl for citationberg::Text {
                 {
                     render_typed_num(num.as_ref(), NumberForm::default(), var, None, ctx);
                 }
+                NumberVariableResult::Regular(n)
+                    if matches!(var, NumberVariable::Page) =>
+                {
+                    ctx.push_str(&n.to_str().replace("-", "–"))
```
Comment (reknih) [inline src/csl/rendering/mod.rs:91]: Does the CSL standard hard-code the en-dash here?

Comment (DerDrodt) [reply src/csl/rendering/mod.rs:92]: Not really. Honestly, this is a quick fix so we can handle these cases correctly (for the most part). I will work rework the page ranges in general. They should be able to handle more than numbers. I should get around to that some time next week.

Comment (reknih) [reply src/csl/rendering/mod.rs:92]: Got it

**Comment by reknih** [comment]
Thanks!

By the way, are you and @Drodt the same person?

**Comment by DerDrodt** [comment]
> Thanks!
> 
> By the way, are you and @Drodt the same person?

You're welcome! And yes. @Drodt is my work account and when I commit stuff while traveling it comes from my work laptop. Sometimes I forget to swap. I hope it's not too confusing!

**Pull request merged**